### PR TITLE
refactor: article image

### PR DESCRIPTION
**Pull request by colincclark**
- refactor article image
- increase coverage to 100%
- add readme

### CODE REVIEW

File: packages/article-image/__tests__/web/__snapshots__/article-image.web.test.js.snap [hunk 1]
```diff
@@ -4,7 +4,25 @@ exports[`1. does not render an image if display is not received 1`] = `Array []`
 
 exports[`2. does not render an image if ratio is not received 1`] = `Array []`;
 
-exports[`3. image with no caption 1`] = `
+exports[`3. does not render an inline image if ratio is not received 1`] = `
+<InlineImage
```
Comment (craigbilner) [inline packages/article-image/__tests__/web/__snapshots__/article-image.web.test.js.snap]: the snapshot and title don't seem to line up?

File: packages/article-image/__tests__/web/__snapshots__/article-image-with-style.web.test.js.snap [hunk 2]
```diff
@@ -19,20 +19,22 @@ Array [
     />
   </div>,
   <div>
-    <div
-      className="c0"
-    >
-      <div>
+    <InsetCaptionStyle>
```
Comment (craigbilner) [inline packages/article-image/__tests__/web/__snapshots__/article-image-with-style.web.test.js.snap]: you need the render serializer to remove the hosts?

File: packages/article-image/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Article content often includes images which are defined in the article image
+package. Article images are configured to be one of several different types
+which dictates the general layout, particularly in relation to their
+accompanying caption.
+
+* primary
+* secondary
+* inline
+
+Ration configuration controls the size of the image, and whether it is portrait
```
Comment (craigbilner) [inline packages/article-image/README.md]: Ratio

File: packages/article-image/README.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Article Image
+
+Article content often includes images which are defined in the article image
+package. Article images are configured to be one of several different types
+which dictates the general layout, particularly in relation to their
+accompanying caption.
+
+* primary
```
Comment (craigbilner) [inline packages/article-image/README.md]: would be nice to get a sense of what these are?

File: packages/article-image/README.md (hunk 3 above)
Comment (colincclark) [reply packages/article-image/README.md]: done

File: packages/article-image/README.md (hunk 4 above)
Comment (colincclark) [reply packages/article-image/README.md]: done

File: packages/article-image/__tests__/web/__snapshots__/article-image.web.test.js.snap (hunk 1 above)
Comment (colincclark) [reply packages/article-image/__tests__/web/__snapshots__/article-image.web.test.js.snap]: done

File: packages/article-image/__tests__/web/__snapshots__/article-image-with-style.web.test.js.snap (hunk 2 above)
Comment (colincclark) [reply packages/article-image/__tests__/web/__snapshots__/article-image-with-style.web.test.js.snap]: done

**Pull request merged**